### PR TITLE
html and css features of the project

### PR DESCRIPTION
**Pull request by BEKMAL**
html and css features added to the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```
Comment (danmainah) [inline README.md]: - [x] Kindly use the microverse [template](https://github.com/microverseinc/readme-template) to create your readme and ensure its neat and professional

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (danmainah) [inline test.md:1]: - [x] Kindly make sure this file is ignored by the .gitignore file, it seems there is an error, try to redo the process.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @BEKMAL , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used



# Required Changes ♻️

- [x] Kindly make sure to add the pull request description, also change your pull request title to be more descriptive on activities done.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (BEKMAL) [reply README.md]: thank you for your comment

File: test.md (hunk 2 above)
Comment (BEKMAL) [reply test.md:1]: ok done

Comment (BEKMAL) [reply test.md:1]: after i have done the comment, is it expected to send another review request?

**Comment by BEKMAL** [comment]
thank you so much,can you re-review now

**Review by OLIPLICHE — APPROVED** [review]
Hi @BEKMAL , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BEKMAL** [comment]
thank you bro

**Pull request merged**